code/modules/research/designs/autolathe_designs.dm

```diff
@@ -561,6 +561,14 @@
 	category = list("initial", "Medical", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
 
+/datum/design/hacksaw
+	name = "Hacksaw"
+	id = "hacksaw"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron=9000, /datum/material/glass=3000)
```

Comment:
Why does the hacksaw take glass to produce?

Reply:
IIRC I based the cost of the hacksaw off of the circular saw, tbh it does make more sense for it to be just pure iron instead of iron and glass. I'll tweak that to be 12000 iron instead of 9000 iron and 3000 glass when I get a chance.

Reply:
Updated to be 12000 iron instead.

Reply:
👍